.github/workflows/manual-deploy-testing.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: Manual Deploy to Testing
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: tag
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: hathitrust/github_actions/build@v1
+        with:
+          dockerfile: Dockerfile.prod
+          image: ghcr.io/${{ github.repository }}-unstable
+          tag: ${{ github.event.inputs.tag }}
+          push_latest: false
+          registry_token: ${{ github.token }}
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    environment: testing
+    steps:
+      - name: Clone latest repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Find commit for tag
+        id: tag_check
+        uses: hathitrust/github_actions/validate-tag@v1
+        with:
+          tag: ${{ github.event.inputs.tag }}
+      - name: Deploy to testing
+        uses: mlibrary/deploy-to-kubernetes@v3
+        with:
+          registry_token: ${{ github.token }}
```

Comment:
I think you do have something in testing? But this won't update the cron job there (the version you have for production will, since you set resource type cronjob, and set the resource & container name, etc)